app/src/main/res/layout/activity_movie_details.xml

```diff
@@ -4,39 +4,293 @@
-        android:layout_alignParentStart="true"
-        android:layout_alignParentRight="true"
-        android:layout_alignParentEnd="true" />
+    tools:context=".MovieDetailsActivity"
+    android:id="@+id/movieDetailsRL">
+
+    <ScrollView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:layout_marginBottom="0dp">
+
+        <RelativeLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:orientation="vertical">
+
+            <View
+                android:id="@+id/rectangle_at_the_top"
+                android:layout_width="match_parent"
+                android:layout_height="10dp"
+                android:layout_alignStart="@+id/tvTitle"
+                android:layout_alignTop="@+id/tvTitle"
+                android:layout_alignEnd="@+id/tvTitle"
+                android:layout_marginTop="20dp"
+                android:layout_marginRight="-10dp"
+                android:background="#4d4e4f"
+                android:visibility="visible" />
+
+            <TextView
+                android:id="@+id/tvTitle"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_below="@+id/relative_layout"
+                android:layout_marginStart="5dp"
+                android:layout_toEndOf="@+id/ivPoster"
+                android:textSize="24sp"
+                tools:text="Movie Title" />
+
+            <TextView
+                android:id="@+id/tvRelease"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_below="@+id/tvTitle"
+                android:layout_alignStart="@+id/tvTitle"
+                tools:text="Release Date: "
+                android:layout_marginTop="5dp"/>
+
+            <RatingBar
+                android:id="@+id/rbVoteAverage"
+                style="@style/Widget.AppCompat.RatingBar.Small"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_below="@+id/tvRelease"
+                android:layout_alignStart="@+id/tvTitle"
+                android:layout_marginTop="5dp"
+                android:numStars="5"
+                android:stepSize="0.5" />
+
+            <TextView
+                android:id="@+id/tvOverview"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_below="@+id/rbVoteAverage"
+                android:layout_alignStart="@+id/tvTitle"
+                android:layout_alignParentEnd="true"
+                android:layout_alignParentRight="true"
+                android:layout_marginTop="6dp"
+                android:text="TextView" />
+
+            <RelativeLayout
+                android:id="@+id/relative_layout"
```

Comment:
you have multiple relative layouts here, so this id may not be as specific as needed. The naming convention should also be snake case